src/app/create-event/create-event.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit, ViewChild, Injectable, ViewChildren } from "@angular/core";
+import { HttpClient } from "@angular/common/http";
+import { NgForOf } from '@angular/common';
+import { NgForm } from '@angular/forms';
+
+@Component({
+  selector: "create-event",
+  templateUrl: "./create-event.component.html",
+  styleUrls: ["./create-event.component.css"]
+})
+export class CreateEventComponent implements OnInit {
+  @ViewChild('f', { static: false }) eventForm: NgForm;
+  posts: any[];
+  private url = "http://jsonplaceholder.typicode.com/posts";
+
+  constructor(private http: HttpClient) { }
+
+  ngOnInit() {
+    this.posts = [];
+  }
+
+  onSubmit(f) {
+    let post = { event: f };
+    this.http.post(this.url, post).subscribe(response => {
+      console.log("Success!", response);
```

Comment:
I would say logging to the console is a good way to debug while you are working on developing, but shouldn't necessarily be kept in the code long term. Think about what you want a user to see if they opened developer tools

Reply:
There's some really great loggers you can find around the internet, which is how you'd want to log things if you still needed some sort of error tracking into production. Don't worry about that for this project though, just wanted you to be aware! popular logger: https://logging.apache.org/log4j/2.x/

Reply:
Thanks guys, I will definitely incorporate those changes.